reportportal/templates/api-deployment.yaml

```diff
@@ -39,7 +39,7 @@ spec:
           value: "{{ .Values.serviceapi.jvmArgs }}"
         {{- end }}
         - name: RP_AMQP_HOST
-          value: "{{ .Values.rabbitmq.endpoint.address }}"
+          value: "{{ .Values.rabbitmq.endpoint.address | default (printf "%s-rabbitmq.%s.svc.cluster.local" .Release.Name .Release.Namespace) }}"
```

Comment:
Unfortunately this is not how the endpoint address usually generated.
For "rabbitmq" release name it will be `rabbitmq.namespace.svc.cluster.local` and for other values it will be as you described. So I don't see an option to generate default value here, since it will be a source of misunderstanding.

Reply:
Hi @HardNorth , 

From the principle that the default "installation" is using the helm dependencies by enabling them (eg. : https://github.com/reportportal/kubernetes/blob/develop/reportportal/values.yaml#L234 ) , the default endpoint of rabbit should be : <Release.Name>-rabbitmq.<Release.Namespace>.svc.cluster.local

The installation of ReportPortal in my minikube confirm this assumption.